packages/reactivity/__tests__/effectScope.spec.ts

```diff
@@ -222,7 +222,7 @@ describe('reactivity/effect/scope', () => {
 
   it('should dereference child scope from parent scope after stopping child scope (no memleaks)', () => {
     const parent = new EffectScope()
-    const child = parent.run(() => new EffectScope())!
+    const [child] = parent.run(() => [new EffectScope(), new EffectScope()])!
```

Comment:
I think it'd be better to add a new test for this, rather than modifying the existing test.

In `effectScope.ts`, there's this `if` check:

```js
if (last && last !== this) {
```

That line is only hit once by the tests. Currently it evaluates to `false`. The change proposed here would switch that, so it evaluates to `true`. That increases the line coverage, but it's still only testing one of the two branches of the `if`. Either way there's a chunk of code that could be removed and the tests would still pass.

For example, imagine that `if` check were changed to this:

```js
if (last) {
```

That would be incorrect and the existing test would catch that, but the new test would still pass.